core/src/main/java/org/elasticsearch/tribe/TribeService.java

```diff
@@ -365,8 +365,13 @@ public String describeTasks(List<ClusterChangedEvent> tasks) {
         }
 
         @Override
-        public ClusterTaskResult<ClusterChangedEvent> execute(ClusterState currentState, List<ClusterChangedEvent> tasks) throws Exception {
-            ClusterTaskResult.Builder<ClusterChangedEvent> builder = ClusterTaskResult.builder();
+        public boolean runOnlyOnMaster() {
```

Comment:
They did not pass before rolling this back. I must have pushed something different to what I tested (it was not even compiling).